internal/cloudapi/v2/server.go

```diff
@@ -84,6 +84,7 @@ func (s *Server) Handler(path string) http.Handler {
 	e.Binder = binder{}
 	e.HTTPErrorHandler = s.HTTPErrorHandler
 	e.Pre(common.OperationIDMiddleware)
+	e.Use(middleware.BodyLimit("1MB"))
```

Comment:
Tbh I think we should enforce this limit on the level of the gateway (and there is a limit there, though it's a lot bigger than 1MB).

Reply:
Cool, I removed it, thanks